hera_cal/smooth_cal.py

```diff
@@ -324,24 +324,27 @@ def narrow_refant_candidates(candidates):
     return refant
 
 
-def rephase_to_refant(gains, refant, flags=None):
+def rephase_to_refant(gains, refant, flags=None, propagate_refant_flags=False):
     '''Rephase all gains in place so that the phase of the reference antenna is 0 for all times and frequencies.
 
     Arguments:
         gains: Dictionary mapping antenna keys to gain waterfalls. Modified in place.
         refant: Antenna key of antenna to make the reference antenna (or dictionary mapping pols to keys)
         flags: Optional dictionary mapping antenna keys to flag waterfall.
             Used only to verify that all gains are flagged where the refant is flagged.
+        propagate_refant_flags: If True and flags is not None, update flags so that all antennas are flagged
```

Comment:
expand this a tiny bit; "...all antennas are flagged at the time/frequencies where the reference antenna was also flagged."

Reply:
Done.